nova_vm/src/ecmascript/builtins/indexed_collections/typed_array_objects/typed_array_intrinsic_object.rs

```diff
@@ -3150,23 +3175,16 @@ fn filter_typed_array<'a, T: Viewable>(
 }
 
 fn copy_between_different_type_typed_arrays<Src: Viewable, Dst: Viewable>(
-    kept: &[Src],
-    byte_slice: &mut [u8],
+    src_slice: &[Src],
+    dst_slice: &mut [Dst],
```

Comment:
By changing the arguments of this method to an immutable T and a mutable V, it has made it possible to reuse the internal processing of this method in other contexts.
What do you think?